### ISSUE TITLE
Meetup - Polkadot - Barcelona, Spain

### ISSUE THREAD

**Issue by EdwardAThomson** [w3f/General-Grants-Program#17]
We are looking to invite people to help organise and co-host a Polkadot meetup in Barcelona, Spain.

**Comment by adriamb** [comment]
Hi @EdwardAThomson I'm the organizer of the Barcelona Ethereum Developers Meetup (https://www.meetup.com/ethereumbcn), and I would love organizing a workshop on parity's tech. It's a meetup for developers, so the idea is to understand how to develop over existing technologies, more than just "set up a node". Since you're using rust, I think that could be usefull to do a workshop about rust just before (it's not well known by the community and it's a must), it would help people to have a some kind of kickoff about starting to use parity's tech. Polkadot is really nice but I think that substrate could be a really, really interesting.

**Comment by EdwardAThomson** [comment]
Hi @adriamb That's great to hear! I think there is interest in doing a technical deep dive as you describe. Someone in Western Switzerland just asked us about something very similar. At the moment we probably don't have a workshop ready to roll out but I think we as are getting more requests for this then it is something we should start to build out.

Let me chat on this with my colleagues and see what we can figure out.

One thing that really helps us to coordinate is the coordination forms we have for meetups. It helps us to gather email addresses privately (etc):

[Collaboration form](https://docs.google.com/forms/d/e/1FAIpQLSe-hbgzMHkwYLCy9-0NLmnpIJdD-nwq_vqNwba9JiiKFmZmZg/viewform)

**Comment by EdwardAThomson** [comment]
The following comment was posted by @Ferdinandreyes in the issue referenced above (#36).

"
I am Ferran, from Caelum Labs, a Barcelona-based company designing and developing Blockchain solutions.
In parallel with that, we dynamize the blockchain community organizing Devcentralised meetup, with almost 600 members at this moment.

We would like to host one Polkadot Meetup in Barcelona, focused on the technology and for devs.

**How many people**: We count on one person to organize the meetup but also a team that can help with it.

**Venue**: We have space in our office to organize the meetup but we can find other places if necessary.

**Do you need help from W3F**: We would need a team member to do a presentation of the technology behind Polkadot and able to solve tech doubts from the community. We can organize it also as a workshop or focused in some of the polkadot network features.

**Do you need help from the community**: Would be nice to have promotion from Polkadot community.
"

**Comment by ljg583** [comment]
Hello! If this thread is still active, please email me at Lindsay@web3.foundation as we would love to support and help grow the Barcelona community!!

**Comment by adriamb** [comment]
Hi @ljg583 , yes, it is still active.
